go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -83,7 +83,7 @@ func (oc *Controller) addEgressFirewall(egressFirewall *egressfirewallapi.Egress
 	var errList []error
 	for i, egressFirewallRule := range egressFirewall.Spec.Egress {
 		//process Rules into egressFirewallRules for egressFirewall struct
-		if i > defaultStartPriority {
+		if i > egressFirewallStartPriority-interNodePolicyPriority {
```

Comment:
I would just make a constant that is like minimumReservedPolicyPriority and make it like 2000. interNodePolicyPriority is not the highest priority today (1003), we use 1005 for mgmtPortPolicyPriority. We also may add more policies in the future, so better to build some buffer room.

Reply:
done